shard/src/test/resources/application.properties

```diff
@@ -0,0 +1 @@
+quarkus.scheduler.enabled=false
```

Comment:
@danielezonca The scheduled methods (for example the job that fetches the Bridges to deploy every 30s) are tested with specific tests, the quarkus application should not start them otherwise they might mess up the interactions with the mocks/update the status of the objects without control